sqlglot/expressions.py

```diff
@@ -4274,6 +4274,7 @@ class Rollback(Expression):
 class AlterTable(Expression):
     arg_types = {
         "this": True,
+        "kind": True,
```

Comment:
Since this is now expecting a `kind` argument, so I'd rename the expression to `Alter` because `AlterTable` is a misnomer when the `kind` is not `"TABLE"`.

@tobymao FYI this is gonna be a breaking change, e.g. every `AlterTable` in SQLMesh is affected.

Reply:
I can rename to `Alter`, and that would touch lots of places since `AlterTable` is used widely.

Reply:
Renamed.